charts/aws-ebs-csi-driver/templates/node-windows.yaml

```diff
@@ -58,7 +58,7 @@ spec:
             - --volume-attach-limit={{ .Values.volumeAttachLimit }}
             {{- end }}
             - --logtostderr
-            - --v=5
+            - --v={{ .Values.nodeWindows.logLevel }}
```

Comment:
The windows node currently uses the `node.*` values so this should be changed to match